plugins/woocommerce/changelog/fix-32754-inactive-downloads

```diff
@@ -0,0 +1,5 @@
+Significance: patch
+Type: tweak
```

Comment:
I think this is the proper way to handle that scenario. Internal link: pcShBQ-fw-p2#comment-691

We still add the changelog file (otherwise the linter complains) and use it to document why we felt no changelog is necessary (via the comment field), but omit any actual changelog text.